zerver/webhooks/github/view.py

```diff
@@ -910,7 +911,17 @@ def api_github_webhook(
     payload: JsonBodyPayload[WildValue],
     branches: str | None = None,
     user_specified_topic: OptionalUserSpecifiedTopicStr = None,
+    skip_private_repositories: Json[bool] = False,
```

Comment:
We've been using the word "ignored" a lot in this integration, so I think something like `ignore_private_repositories` feels more consistent.